src/Commands/CleanCategory.php

```diff
@@ -0,0 +1,238 @@
+            ->setHelp('This command :'
+                . PHP_EOL . '   - Enable or disable a category.'
+                . PHP_EOL . '   - Disable final categories without product.'
+                . PHP_EOL . '   - Enable final categories with an active product.'
+                . PHP_EOL . '   - This command DON\'T SUPPORT multi-shop.')
+            ->addUsage('--exclude=[XX,YY,ZZ] (id_category separate by coma)')
+            ->addArgument(
+                'action',
+                InputArgument::OPTIONAL,
+                'Disable, Enable, Disable empty categories or Enable no empty categories ( possible values : ' . $this->getPossibleActions() . ') ',
+                'status'
+            )
+            ->addOption('id_lang', null, InputOption::VALUE_OPTIONAL, 'Id lang')
+            ->addOption('id_category', null, InputOption::VALUE_OPTIONAL)
+            ->addOption('exclude', null, InputOption::VALUE_OPTIONAL, 'Ids Category to exclude')
+            ->addOption('force', 'f', InputOption::VALUE_NONE, 'Force the command when the MultiShop is enable.');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+        $force = $input->getOption('force');
+
+        if (1 < Shop::getTotalShops(false)) {
+            if (!$force) {
+                $io->error('Actualy this command don\'t work with MultiShop.'
+                . PHP_EOL . 'Use force (-f) option to run the command.');
+
+                return 1;
+            } else {
+                $io->warning('MultiShop Enable, Force Mode');
+            }
+        }
+
+        $action = $input->getArgument('action');
+        $id_lang = $input->getOption('id_lang') ?? Configuration::get('PS_LANG_DEFAULT');
+        $exclude = $input->getOption('exclude') ? explode(',', $input->getOption('exclude')) : [];
+
+        switch ($action) {
+            case 'status':
+                $categories = $this->getCategoriesToClean($id_lang, $action, $exclude);
+
+                if (!$categories['empty'] && !$categories['noempty']) {
+                    $io->title('No Categories to updated');
+
+                    return 0;
+                }
+
+                if ($categories['empty']) {
+                    $io->title('The following categorie(s) are enabled but without product active');
+                    $io->text(implode(' / ', $categories['empty']));
+                    $io->text(' -- You can run `./bin/console fop:category disable-empty` to fix it');
+                    $io->text(' -- If you want exclude categories you can add --exclude ID,ID2,ID3');
+                }
+
+                if ($categories['noempty']) {
+                    $io->title('The following categorie(s) are disabled but with product active in the category');
+                    $io->text(implode(' / ', $categories['noempty']));
```

Comment:
What about giving the id of the category in addition with this name here ?
It could be usefull when we have several categories with the same name.
![image](https://user-images.githubusercontent.com/7163132/120233685-ac61d200-c256-11eb-8473-e3eb600a2991.png)
WDYT ?

Reply:
Thank you for feedback @nenes25 i add this